pkg/koordlet/metrics/resctrl.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"strconv"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	ResourceTypeLLC = "llc"
+	ResourceTypeMB  = "mb"
+
+	ResctrlResourceType = "resource_type"
+	ResctrlCacheId      = "cache_id"
+	ResctrlQos          = "qos"
+	ResctrlMbType       = "mb_type"
+)
+
+var (
+	QosResctrl = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Subsystem: KoordletSubsystem,
+		Name:      "qos_resctrl",
```

Comment:
"qos" is already included in labels, maybe define as two metric such as "llc_occupancy" and "mbm_occupancy". so that we don't need to set "MetricPropertyResctrlMbType="") during record metrics.

so does the `MetricPropertiesFunc`